.github/workflows/publish-packages.yml

```diff
@@ -183,7 +183,7 @@ jobs:
 
       - name: Install Required tools
         run: |
-          yum install -y maven rpm-sign rpm-build wget
+          yum install -y maven rpm-sign rpm-build wget gettext
```

Comment:
Is this related or by mistake?

Reply:
it's related: after the upgrade `envsubst` command is not available. It's part of the `gettext` package